src/components/two-factor-authentication/SoftwareTokenWorkflow.spec.ts

```diff
@@ -1,33 +1,59 @@
 import { mount } from '@vue/test-utils';
 import SoftwareTokenWorkflow from './SoftwareTokenWorkflow.vue';
+import {
+  useTwoFactorAuthentificationStore,
+  type TwoFactorAuthentificationStore,
+} from '@/stores/TwoFactorAuthentificationStore';
 
-beforeEach(() => {
-  document.body.innerHTML = `
+let twoFactorAuthenticationStore: TwoFactorAuthentificationStore;
+
+describe('software token workflow', () => {
+  beforeAll(() => {
+    twoFactorAuthenticationStore = useTwoFactorAuthentificationStore();
+  });
+
+  beforeEach(() => {
+    document.body.innerHTML = `
     <div>
       <div id="app"></div>
     </div>
   `;
 
-  mount(SoftwareTokenWorkflow, {
-    attachTo: document.getElementById('app') || '',
-    props: {
-      qrCodeImageBase64: 'data:image/png;base64,abc',
-    },
-    global: {
-      components: {
-        SoftwareTokenWorkflow,
+    mount(SoftwareTokenWorkflow, {
+      attachTo: document.getElementById('app') || '',
+      props: {
+        qrCodeImageBase64: 'data:image/png;base64,abc',
```

Comment:
In Github sieht es so aus, als wäre das falsch eingerückt. Kannst du das bitte mal mit npm run lint gegenprüfen?

Reply:
lint gibt nichts aus also sollte richtig sein